### DISCUSSION TITLE
C++14 digit separators are not recognized correctly

### DISCUSSION THREAD

**Discussion by vinaybs1983** [oracle/opengrok#4160] [General]
In version opengrok-1.1-rc21

As in subject - C++14 introduced single quotes as integer separators:

[https://en.cppreference.com/w/cpp/language/integer_literal]
|Optional single quotes ({{{}'{}}}) may be inserted between the digits as a separator. They are ignored by the compiler.|(since C++14)|

But in OpenGrok it's wrongly recognised as start of single character like

char character = 'a';

**Comment by vladak** [comment]
1.1 is really old. Does this happen with the latest ?
